partner_communication/models/communication_job.py

```diff
@@ -744,9 +739,7 @@ def _send_mail(self):
 
         final_state = "pending"
         if email.state == "sent":
-            if self.need_call == "after_sending":
-                final_state = "call"
-            else:
+            if self.need_call == "before_sending":
                 final_state = "done"
         return final_state
```

Comment:
I think all this code is no more needed, because the final state will always be "done" now. It can be just:
```
return "done"
```

Reply:
Please have a look at this comment as well.

Reply:
Apparently the `final_state` can be pending if the `need_call` is "after_sending". Should I really remove this part and put "done" by default ?

Reply:
yes